spec/controllers/posts_controller_spec.rb

```diff
@@ -1,11 +1,12 @@
 require 'spec_helper'
+require 'pry'
 
 describe PostsController, type: :controller do
   include_context 'JSON API headers'
 
   before(:all) { FactoryGirl.create_list(:post, 3) }
 
-  let(:fields)        { (PostResource.fetchable_fields - %i(id author)).map(&:to_s) }
+  let(:fields)        { (PostResource.new(Post, nil).fetchable_fields - %i(id author)).map(&:to_s) }
```

Comment:
Well, here I guess there is no need of instantiating a new object just to get some meta data like fetchable fields. Since these fields are part of the resource class definition, it makes more sense to get them from a class method.